src/bessel.jl

```diff
@@ -131,8 +131,9 @@ for afn in (:airyai, :airyaiprime, :airybi, :airybiprime,
             :airyaix, :airyaiprimex, :airybix, :airybiprimex)
     @eval begin
         $afn(z::Complex) = $afn(float(z))
-        $afn(z::Complex{<:AbstractFloat}) = throw(MethodError($afn,(z,)))
+	$afn(z::Complex{Float16}) = Complex{Float16}($afn(Complex{Float32}(z)))        
```

Comment:
Please preserve the existing indentation (4 spaces per level) and avoid adding trailing whitespace.

Reply:
Done .